cmd/skaffold/app/cmd/fix.go

```diff
@@ -148,3 +156,15 @@ func getOldConfigYaml(cfgs []util.VersionedConfig) string {
 	}
 	return string(yamlStr)
 }
+
+func getTrueLocation(path string) (string, error) {
```

Comment:
I don't think you need this method anymore right?

Reply:
Removed.